composer/algorithms/stochastic_depth/stochastic_layers.py

```diff
@@ -66,18 +66,25 @@ def __init__(self, drop_rate: float, module_id: int, module_count: int, use_same
         self.use_same_depth_across_gpus = use_same_depth_across_gpus
         self.rand_generator = rand_generator
 
+    @torch.jit.unused
+    def _get_sample(self, device: int) -> torch.Tensor:
```

Comment:
We refactor the incompatible code into its own function that is only called during training.

`_sample_bernoulli` is not compatible because the function has an argument of type `torch.Generator`, and torchscript only supports built-ins and `Tensor` types.